edb/schema/types.py

```diff
@@ -2105,7 +2109,14 @@ class CreateCollectionExprAlias(
     CollectionExprAliasCommand[CollectionExprAliasT],
     sd.CreateObject[CollectionExprAliasT],
 ):
-    pass
+    def canonicalize_attributes(
+        self,
+        schema: s_schema.Schema,
+        context: sd.CommandContext,
+    ) -> s_schema.Schema:
+        schema = super().canonicalize_attributes(schema, context)
+        self.set_attribute_value('internal', True)
```

Comment:
It probably makes more sense to set 'internal' in `_handle_alias_op` as the type attribute, and I think this should also be done for non-collection alias types.